test/upgrade/upgrade_test.go

```diff
@@ -127,13 +127,11 @@ func preUpgradeTests() []pkgupgrade.Operation {
 }
 
 func postUpgradeTests(ctx *test.Context) []pkgupgrade.Operation {
-	tests := []pkgupgrade.Operation{
-		waitForServicesReady(ctx),
-		eventingupgrade.PostUpgradeTest(),
-		kafkaupgrade.ChannelPostUpgradeTest(),
+	tests := []pkgupgrade.Operation{waitForServicesReady(ctx)}
+	tests = append(tests, eventingupgrade.PostUpgradeTests()...)
+	tests = append(tests, kafkaupgrade.ChannelPostUpgradeTest(),
 		kafkaupgrade.SourcePostUpgradeTest(),
-		kafkabrokerupgrade.BrokerPostUpgradeTest(),
-	}
+		kafkabrokerupgrade.BrokerPostUpgradeTest())
```

Comment:
reordering the tests is not an issue here, I'd assume. So I'd just delete the PostUpgradeTests from the initial list and append them like the serving tests.

Reply:
I was not sure about order.

Reply:
@mgencur ordering makes a difference here?